<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Año</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="txtAnyo">
       <item>
        <property name="text">
         <string>2017</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Clientes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeClientes">
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4º</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Proveedores</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeProveedores">
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4º</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Gastos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeGastos">
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3º</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4º</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
